src/main/pg/rx.c

```diff
@@ -34,16 +34,6 @@
 #include "rx/rx.h"
 #include "rx/rx_spi.h"
 
-#ifndef SERIALRX_PROVIDER
```

Comment:
Please leave these here. 

Trying to move any "if undefined protection as close to the use of it as possible.

Reply:
Fixed.